docs/sources/guides/whats-new-in-v5-3.md

```diff
@@ -12,6 +12,40 @@ weight = -9
 
 # What's New in Grafana v5.3
 
+Grafana v5.3 brings new features, many enhancements and bug fixes. This article will detail the major new features and enhancements.
+
+- [TV mode]({{< relref "#tv-and-kiosk-mode" >}}) is improved
+- [Alerting]({{< relref "#notification-reminders" >}}) with notification reminders
+- [Postgres query builder]({{< relref "#postgres-query-builder" >}}) it's finally here!
```

Comment:
Unsure about "It's finally here". How about

`[Postgres]({{< relref "#postgres" >}}) with a new graphical query builder`

Reply:
How about, "Postgres gets a new query builder!"